app/assets/javascripts/ico-validator.js

```diff
@@ -33,20 +33,23 @@ jQuery( function($) {
     }
   }
 
-  ico_fields.on('change', function() {
+  $(document).on('change', ico_fields, function(e) {
```

Comment:
Identifier 'ico_fields' is not in camel case.<br>'ico_fields' is not defined.<br>'e' is defined but never used.